mandatory/2-function-creation.js

```diff
@@ -15,26 +21,69 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice > 200){
+    newPrice = totalPrice - (0.1 * totalPrice);
+
+  }
+  else{
+    newPrice = totalPrice - (0.05 * totalPrice);
+  }
+ return newPrice;
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(limit) {
+  number = 0;
+  while(number < limit){
+    number = number + 1;
+    if (number === 1){
+    console.log(number);
+  }else if(number % 2 !== 0){
+    console.log(number)
+  }
+    
+
+  }
+}
 
 /*
   Complete the buyTwoGetTheCheapestFree function: if user buys two items, the cheapest item will be free!
   The function should return the price to be paid once the discount is applied.
 */
-function buyTwoGetTheCheapestFree(price1, price2) {}
+function buyTwoGetTheCheapestFree(price1, price2) {
+  if (price1 === price2){
+    return price1 + price2;
+  
+  }else if(price1 < price2){
+    return price2;
+
+  }
+  else{
+    return price1;
+  }
+
+}
```

Comment:
As discussed in the buddy session. We can use something called a [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) here to make this super short.
```
function buyTwoGetTheCheapestFree(price1, price2) {
 return price1 < price2 ? price2 : price1
}
```
This works since if the prices are the same then it will just return one of them (doesn't matter which), in our case, `price1 < price2` will not be true if they are the same, so our ternary operator will return what's after the `:`, which is `price1`